template/src/pages/buy-me-coffee/MemoCard.tsx

```diff
@@ -11,27 +11,36 @@ import type { CoffeeMemo } from '../../types';
   const convertedTimestamp = convertBigIntTimestampToDate(time);
   return (
-    <div className="rounded-xl border border-solid border-zinc-700 bg-zinc-900 p-3">
-      <div className="flex justify-start gap-3">
-        <div className="flex items-center justify-between gap-3">
-          <div className="relative inline-flex h-10 w-10">
-            <span className="flex h-full w-full items-center justify-center rounded-full bg-gray-800">
-              <span className="text-sm font-medium uppercase text-gray-700 dark:text-gray-400">
-                {userName[0]}
-              </span>
-            </span>
+    <div className="relative flex w-full flex-[0_0_auto] flex-col items-start gap-[16px] self-stretch">
+      <div className="relative flex w-full flex-[0_0_auto] items-center justify-between self-stretch">
+        <div className="relative inline-flex flex-[0_0_auto] items-center gap-[16px]">
+          <div className="relative h-[40px] w-[40px] rounded-[20px] bg-boat-color-yellow-60">
+            <div className="absolute left-[15px] top-[9px] whitespace-nowrap text-center text-[16px] font-semibold leading-[normal] tracking-[0] text-white [font-family:'Inter-SemiBold',Helvetica]">
+              {userName[0]}
+            </div>
           </div>
+          <div className="relative inline-flex flex-[0_0_auto] items-start gap-[4px]">
+            <div className="relative inline-flex flex-[0_0_auto] items-center gap-[8px]">
+              <div className="relative mt-[-1.00px] w-fit whitespace-nowrap text-[16px] font-bold leading-[24px] tracking-[0] text-boat-color-palette-foreground [font-family:'Inter-Bold',Helvetica]">
+                {userName}
+              </div>
+            </div>
+            <div className="relative mt-[-1.00px] w-fit whitespace-nowrap text-[16px] font-normal leading-[24px] tracking-[0] text-boat-color-palette-foregroundmuted [font-family:'Inter-Regular',Helvetica]">
+              bought 1 coffee
+            </div>
+          </div>
+        </div>
+        <div className="relative w-fit whitespace-nowrap text-[16px] font-normal leading-[24px] tracking-[0] text-boat-color-palette-foregroundmuted [font-family:'Inter-Regular',Helvetica]">
+          {convertedTimestamp.toDateString()}
         </div>
-        <div className="grow-1 flex flex-col justify-start gap-2">
-          <div className="flex items-center justify-between">
-            <span className="mr-2 text-sm font-bold">{userName}</span>
-            <span className="text-sm font-normal text-gray-400">
-              {convertedTimestamp.toDateString()}
-            </span>
+      </div>
+      <div className="relative flex w-full flex-[0_0_auto] items-center gap-[273px] self-stretch rounded-[16px] border-2 border-solid border-transparent bg-[#161616e6] p-[24px] backdrop-blur-2xl backdrop-brightness-[100%] [-webkit-backdrop-filter:blur(40px)_brightness(100%)] [border-image:linear-gradient(to_bottom,rgb(245,89,37),rgb(215,89,134))_1]">
```

Comment:
When tailwind gets too long, please use `clx` to split in two/three lines. It makes it way eaiser to read.